.travis.yml

```diff
@@ -2,6 +2,18 @@ language: python
 
 sudo: required
 
+# if the https://travis-ci.org/ODM2/ODM2PythonAPI/requests ever says: missing config
```

Comment:
@miguelcleon Any reason why these were added? It seems like your changes may have broken Travis CI.

Reply:
Any changes to Travis should go through a PR so it doesn't just break without anyone knowing why. Thanks. I am going to roll back your changes to fix Travis.

Reply:
Your changes are now in PR, please remove any change in `.travis.yml`

Reply:
Yes this looks like a change I didn't mean to include.